xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -31,6 +32,10 @@
 #include "utils/StringUtils.h"
 #include "utils/URIUtils.h"
 
+#ifdef TARGET_POSIX
+#include "platform/posix/XTimeUtils.h"
+#endif
```

Comment:
Is this new include actually needed?

Reply:
I could try removing it and see does it pass a build. Maybe after Milhouse has his build done.

Reply:
I'm pretty sure it will build without this new include. ;-)

Reply:
Yep, it's thew Sleep(10); which requires the new include. Sorry for the noise.

=> https://github.com/xbmc/xbmc/blob/master/xbmc/platform/posix/XTimeUtils.h#L15

Reply:
Yep, I had just finished building myself ;)